packages/fx-core/src/plugins/resource/frontend/resources/messages.ts

```diff
@@ -30,10 +33,14 @@ export class Messages {
   static readonly EndPreDeploy = (name: string) => `Pre-deployed '${name}'.`;
   static readonly StartDeploy = (name: string) => `Deploying '${name}'.`;
   static readonly EndDeploy = (name: string) => `Successfully deployed '${name}'.`;
-  static readonly StartPostLocalDebug = (name: string) => `Starting local debug '${name}'.`;
```

Comment:
Is this "StartPostLocalDebug" not used anymore?

Reply:
Yes. We don't have post local debug lifecycle in frontend plugin anymore.